etl/extract.py

```diff
@@ -1,12 +1,24 @@
 """
 Functions to retrieve data from the Well Registry API
 """
-import requests as r
+
+from requests import Session
 
 
 def get_monitoring_locations(registry_ml_endpoint):
     """
     Get the monitoring location data.
     """
-    resp = r.get(registry_ml_endpoint)
-    return resp.json()
+    next_chunk = registry_ml_endpoint
+    results = []
+
+    with Session() as session:
+        while next_chunk:
+            print(f'Retrieving monitoring locations: {next_chunk}')
+            resp = session.get(next_chunk)
+            payload = resp.json()
+            results.extend(payload['results'])
+            next_chunk = payload['next']
+
+    print(f'Finished retrieving {payload["count"]} monitoring locations.')
```

Comment:
Should this be using a logger instead?

Reply:
I had logger.info in here and could put it in again. I'll put it back and set the level to INFO.